client/packages/admin/src/index.ts

```diff
@@ -367,11 +370,15 @@ class InstantAdmin<
    */
   debugQuery = async <Q extends Query>(
     query: Exactly<Query, Q>,
-    opts?: { rules: any },
+    opts?: { rules?: any; ruleParams?: { [key: string]: any } },
   ): Promise<{
     result: QueryResponse<Q, Schema, WithCardinalityInference>;
     checkResults: DebugCheckResult[];
   }> => {
+    if (query && opts && 'ruleParams' in opts) {
+      query = { $$ruleParams: opts['ruleParams'], ...query };
```

Comment:
I wonder if we should push $$ruleParams into `$`. This way, the backend only needs to worry about ruleParams in there. 

It _is_ weird, in that `$` can defined in child queries, and `$$ruleParams` only makes sense at the top level, but since we are the ones inserting the option, perhaps not a big deal. 

Am open to either way, but raising in case I am missing something

Reply:
The idea here was that query can have multiple namespaces (can it?) like `{docs: {}, comments: {}}`, but we only define `ruleParams` once for the whole query. But if we only allow single top-level ns, then for sure we should move it inside $

Reply:
Ah, good point! Forgot about multiple namespaces case